libqsopt/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+	<name>libqsopt</name>
+	<version>0.6.10</version>
+	<description>Wrapper for the Qsopt linear programming solver</description>
+
+	<url type="website">http://www.math.uwaterloo.ca/~bico/qsopt/index.html</url>
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+
+	<license>GPL</license>
```

Comment:
better. But then again: this is redistributing something that you are not sure of what the license is....